lib/generate-doc.js

```diff
@@ -29,7 +29,11 @@ module.exports = function generateDocument (baseDocument, router, basePath) {
       if (routeLayer && routeLayer.keys && routeLayer.keys.length) {
         const keys = {}
 
-        const params = routeLayer.keys.map((k) => {
+        const params = routeLayer.keys.map((k, i) => {
+          const prev = routeLayer.keys[i - 1]
+          if (!Number.isNaN(k.name) && prev?.offset + prev?.name?.length - 2 === k.offset) {
```

Comment:
I had to read this line a few times to make sure I followed, might be good to just add a comment about it for our future selves.

Reply:
I am testing this line to make sure it does not have any bugs.

Reply:
@wesleytodd I am done double checking for bugs, please re-approve and merge at your convenience.